spec/requests/v1/items_request_spec.rb

```diff
@@ -93,4 +89,19 @@
         expect(item.unit_price).to_not eq(previous_price)
         expect(item.unit_price).to eq(80.25)
     end
+
+    it "can destroy an item" do
+        item = Item.create(name: "REGRET",
+            description: "Hard work rarely pays off.",
+            price: 69.00,
+            merchant_id: @merchant.id)
+        
+        expect(Item.count).to eq(5)
+    
+        delete "/api/v1/items/#{item.id}"
+    
+        expect(response).to be_successful
+        expect(Item.count).to eq(10)
```

Comment:
I'm not sure why the `Item.count` value goes from `5` before the delete, and becomes `10`.  I would like to talk about this in more detail, if you find time can you elaborate on this decision?